src/Velociraptor.js

```diff
@@ -106,6 +110,9 @@ module.exports = React.createClass({
             });
 
             //console.log("enter: "+JSON.stringify(node.props._restingStyle));
+        } else if (node === null) {
+            // https://github.com/julianshapiro/velocity/issues/300
+            Velocity.Utilities.removeData(this.nodes[k]);
```

Comment:
This looks plausible to me, but it's far from obvious from reading the linked bug. Could you add a more verbose comment - something along the lines of "Tell the Velocity jquery shim to forget about any data associated with this node"